pkg/helm/chartproxy/proxy_test.go

```diff
@@ -44,7 +74,7 @@ func TestProxy_IndexFile(t *testing.T) {
 		{
 			name:       "returned merged index file for all accessible helm repos",
 			indexFiles: []string{"testdata/azureRepoIndex.yaml"},
-			mergedFile: "testdata/azureRepoIndex.yaml",
+			mergedFile: "testdata/mergedAzureRepoIndex.yaml",
```

Comment:
revert this change now, give that we have separate PR #7111

Reply:
Done